README.md

```diff
@@ -18,12 +18,9 @@ Grab the [package from NuGet](https://nuget.org/packages/DogStatsD-CSharp-Client
 ## Platforms
 
 DogStatsD-CSharp-Client supports the following platforms:
-* .NET Standard 1.3
-* .NET Standard 1.6
-* .NET Core Application 1.1
-* .NET Core Application 2.0
-* .NET Framework 4.5.1
-* .NET Framework 4.6.1
+* .NET Standard 1.3 or greater
```

Comment:
> According to https://docs.microsoft.com/en-us/dotnet/standard/net-standard the .Net Framework 4.5.1 does not implement the .NET Standard 1.3

Indeed, that is why I wrote that it also support `.NET Framework 4.5.1 or greater`.

I list explicitly what is supported so that user does not have to read the compatibility table.

Note: we cannot use `.Net Standard 1.2` because it does not support Socket class for example (https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.socket?view=netframework-4.8)